index.html

```diff
@@ -560,15 +564,18 @@
         <div dfn-for="MediaTrackSettings">
           <p>
             The <dfn><code>focalLength</code></dfn> dictionary member
-            represents the <a>depth map</a>'s <a>focal length</a>.
+            represents the <a>depth map</a>'s <a>focal length</a>. It is a
+            <a>non-constrainable property</a>.
```

Comment:
`non-constrainable` seems like a misnomer. Constraints are properties of the *application*, as in "here are my constraints", a request. So it is perfectly reasonable to set constraints on inherent unchangeable properties. Remember `getUserMedia` may pick from several cameras/mics.